embc-app/ClientApp/src/app/volunteer-dashboard/volunteer-dashboard.component.html

```diff
@@ -72,7 +72,7 @@ <h3 class="blue-header">LIST OF EVACUEES</h3>
   <div class="col-12 col-lg-4">
     <app-side-box-info title="ADD AN EVACUEE"
       content="Before adding volunteers, confirm that they have been registered by the Emergency Social Services Director."
```

Comment:
Please check the content on the wireframes. The link to add evacuees and content of the box ("add more volunteers") do not match.

Reply:
Ah yes. I had built both boxes and intended to comment one out. I will fix this in a bit.